core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -272,6 +274,18 @@ object TopicCommand extends Logging {
       val allConfigs = adminClient.describeConfigs(topics.map(new ConfigResource(Type.TOPIC, _)).asJavaCollection).values()
       val liveBrokers = adminClient.describeCluster().nodes().get().asScala.map(_.id())
       val topicDescriptions = adminClient.describeTopics(topics.asJavaCollection).all().get().values().asScala
+      val topicPartitions = topicDescriptions.flatMap(pi => pi.partitions().asScala.map(tpi => new TopicPartition(pi.name(), tpi.partition())))
+      val reassignments: util.Map[TopicPartition, PartitionReassignment] = try {
+        adminClient.listPartitionReassignments(topicPartitions.toSet.asJava).reassignments().get()
```

Comment:
As a potential follow-up KIP, it might be nice to be able to request `--describe --reassigning-partitions`.